### PR TITLE
AB#38676 Cleaned up repo and removed unused and unnecessary files and folders

### PR DESCRIPTION
**Pull request by omersayshi**
### Overview

This repository was full of unnecessary/useless files and required a clean up.

### Testing Instructions
* Just run the app from the root of the repository 
* Make sure everything works as it's should

### CODE REVIEW

**Comment by lhao03** [comment]
Not sure how to test this because this is all I see:

in ToolkitApp:
image

at root:
image

**Comment by ayubun** [comment]
> Not sure how to test this because this is all I see:
> 
> in ToolkitApp:
> image
> 
> at root:
> image

These look like normal screenshots to me; Did you run it in debugger mode? That's what we have all been doing, to my knowledge

**Comment by lhao03** [comment]
Currently the teams app doesn't work in debugger mode. 
image

This is necessary because connected resource, getting RSC permissions belonging to a resource (basically alot of things) depend on the teams app actually being in teams. Can this be refactored such that it still works in debugger mode? 

> > Not sure how to test this because this is all I see:
> > in ToolkitApp:
> > image
> > at root:
> > image
> 
> These look like normal screenshots to me; Did you run it in debugger mode? That's what we have all been doing, to my knowledge

**Review by lhao03 — CHANGES_REQUESTED** [review]
image

why were these files deleted? 

image

File: README.md [hunk 1]
```diff
@@ -1,12 +1,70 @@
+## Which folders are important?
```
Comment (v-mschramm) [inline README.md]: This readme.md file is no longer accurate and cannot be followed.  Please cleanup or remove and indicate the task that has been created to fix this.

**Review by v-mschramm — CHANGES_REQUESTED** [review]

**Comment by ayubun** [comment]
> image
> 
> why were these files deleted?
> 
> image

Ah, I see what you were saying earlier.

Because `settings.json` was deleted, we can no longer run the app in debugger mode, since that file contained all of the relevant info to do such.

I also have the same issue as you; Cannot run/test the app because of such ^^;

**Comment by omersayshi** [comment]
Alot of new stuff was added and the structure of the repository has been discussed again. Closing this PR to open up a new one with simpler reductions to avoid errors.

**Pull request closed without merging**